cmd/kubeadm/app/kubeadm.go

```diff
@@ -35,8 +35,10 @@ func getEnvParams() map[string]string {
 	}
 
 	envParams := map[string]string{
-		"prefix":        globalPrefix,
-		"host_pki_path": path.Join(globalPrefix, "pki"),
+		"prefix":          globalPrefix,
+		"host_pki_path":   path.Join(globalPrefix, "pki"),
+		"hyperkube_image": "gcr.io/google_containers/hyperkube:v1.4.0-alpha.3",
```

Comment:
I'll keep the version hardcoded for the moment, but will make sure to use `runtime.GOARCH` . So, for now, user would have to override the whole image name, I'd rather get this to state the state where we can merge it and work on refining how such parameters are exposed in separate discussions after we merged MVP and people can test by `kubeadm` by checking out master.